CONTRIBUTING.md

```diff
@@ -85,7 +85,7 @@ You can run `tox` with the following arguments:
 * `tox -e generate-workflows` to run creation of new CI workflows if tox environments have been updated
 * `tox -e ruff` to run ruff linter and formatter checks against the entire codebase
 
-`ruff check` and `ruff format` are executed when `tox -e ruff` is run. We strongly recommend you to configure [pre-commit](https://pre-commit.com/) locally to run `ruff` automatically before each commit by installing it as git hooks. You just need to [install pre-commit](https://pre-commit.com/#install) in your environment:
+`ruff check` and `ruff format` are executed when `tox -e ruff` is run. We strongly recommend you to configure [pre-commit](https://pre-commit.com/) locally to run `ruff` and `rstcheck` automatically before each commit by installing it as git hooks. You just need to [install pre-commit](https://pre-commit.com/#install) in your environment:
```

Comment:
Could the bulleted list above at L87 also be updated with a new point about `tox -e rstcheck`?

Reply:
Or, what Aaron suggested [above](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/3777#discussion_r2376357860)!

Reply:
Added tox -e precommit as in core